grammar.js

```diff
@@ -1275,21 +1281,21 @@ module.exports = grammar({
       field('subscript', $.bracketed_argument_list)
     )),
 
-    element_binding_expression: $ => $.bracketed_argument_list,
+    element_binding_expression: $ => field('subscript', $.bracketed_argument_list),
```

Comment:
Perhaps `arguments` is more in line with existing field names?

Reply:
This matches the above `field('subscript', $.bracketed_argument_list)`. For the time being, I'm removing this field name, we can revisit it later...